pouchdb-core/index.d.ts

```diff
@@ -426,6 +431,10 @@ declare namespace PouchDB {
         on(event: 'created', listener: (dbName: string) => any): this;
         on(event: 'destroyed', listener: (dbName: string) => any): this;
 
+        debug: debug.IDebug;
+
+        adapter(id: string, adapter: any, addToPreferredAdapters?: boolean): void;
```

Comment:
You're right, but the way I set it up that method definition is currently contributed by adapter plugins.  Check out an adapter plugin, e.g. `pouchdb-adapter-http/index.d.ts` (I'd provide a proper link, but I'm mobile right now).